hastexo/migrations/0005_auto_20190811_1555.py

```diff
@@ -36,7 +36,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='stacklog',
             name='hook_events',
-            field=jsonfield.fields.JSONField(default=None),
+            field=jsonfield.fields.JSONField(default=b'null'),
```

Comment:
I'm a bit puzzled as to why this is a binary string, but I guess this is from `manage.py makemigrations`?

Reply:
Correct.  Straight from `makemigrations`.